docs/Platforms/cms.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: default
+title: "Tallwave Guidelines — Platforms — Content Websites"
+---
+
+[&larr; Home]({{ site.root }}/) &mdash; [Platforms]({{ site.root }}/platforms)
+
+# Content Based Websites
+
+On today's Internet, content is king. Being the top result on a search engine (Google) provides a significant advantage in the marketplace. [Bill Gates allegedly](https://www.craigbailey.net/content-is-king-by-bill-gates/) said in 1996 "Content is where I expect much of the real money will be made on the Internet, just as it was in broadcasting."
+
+In that essay he goes on to say how it's not just about putting content on the Internet. Rather, it's about *relevant* and *innovative* content. This can take the form of something as small as a single landing page, or a massive content platform serving hundreds, or thousands of pages.
+
+## Marketing Websites
+
+A good marketing website can do so many things for our clients. It's the first thing their customers see so it:
+
+ * Gives the first impression of brand, UX, and product.
+ * If done properly, moves customers through the sales pipeline.
+ * Serves as a portal to other products we may have built.
+ * Demonstrates out abilities at web design and development.
```

Comment:
'Demonstrates out...' is a bit confusing, is this supposed to be 'Demonstrates our...' or 'Serves as a demonstration of web design and development abilities.'?

Reply:
Yup. Should be 'our'.